src/foam/u2/wizard/wao/SplitWAO.js

```diff
@@ -48,9 +48,10 @@ foam.CLASS({
     },
     async function load (wizardlet) {
       if ( wizardlet.loading ) return;
-      wizardlet.loading = true;
       const loader = foam.json.parse(this.loader, undefined, wizardlet.__subContext__);
       foam.u2.wizard.data.ensureTerminal(loader, this.ProxyLoader, this.NullLoader);
+      await this.SUPER(wizardlet);
```

Comment:
Generated proxy implementation's `load` calls the delegate - equivalent to `this.delegate.load` (probably better because then the concern of how to proxy the method belongs to the proxy implementation)